<launch>
    <rosparam file="$(find vawt_hardware_interface)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find vawt_hardware_interface)/config/controllers.yaml" command="load"/>
    <rosparam file="$(find vawt_hardware_interface)/config/joint_limits.yaml" command="load"/>
    <node name="vawt_hardware_interface" pkg="vawt_hardware_interface" type="vawt_hardware_interface_node" output="screen"/>
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
        args="/vawt_1/controller/state
        /vawt_1/controller/position/blade_1_position_controller
        /vawt_1/controller/position/blade_2_position_controller

        "/>
</launch>
